azure/README.md

```diff
@@ -409,10 +409,8 @@ configMapGenerator:
 secretGenerator:
 - name: peer-pods-secret
   namespace: confidential-containers-system
-  literals:
-  - AZURE_CLIENT_ID="${AZURE_CLIENT_ID}"
-  - AZURE_CLIENT_SECRET="${AZURE_CAA_CLIENT_SECRET}"
-  - AZURE_TENANT_ID="${AZURE_TENANT_ID}"
+  envs:
+  - service-principal.env
```

Comment:
/lgtm
nice @mkulke .I'm yet to test though in my setup. Will do it shortly.
Can you please create tracker issues to add a similar approach for the other providers.